src/Phinx/Seed/OrderedSeedInterface.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Phinx
+ *
+ * (The MIT license)
+ * Copyright (c) 2015 Rob Morgan
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated * documentation files (the "Software"), to
+ * deal in the Software without restriction, including without limitation the
+ * rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+ * sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+ * IN THE SOFTWARE.
+ *
+ * @package    Phinx
+ * @subpackage Phinx\Seed
+ */
+namespace Phinx\Seed;
+
+/**
+ * Ordered seed interface
+ * Use this class if you want to define your seeds dependencies
+ * and run seeds in ordered way without using command line options
+ */
+interface OrderedSeedInterface
+{
+    /**
+     * @return array
+     */
+    public function getDependencies();
```

Comment:
Why do we need an additional interface? I think we can add some additional method to the `AbstractSeed` class with a default implementation.

Reply:
Of course we can add method to AbstractSeed. I did interface if some people do not want to order they seeds and then they keep them without this interface and without getDependencies() function.
If there is a need to move to AbstractSeed class I can do it.